packages/ui5-application/test/options.test.ts

```diff
@@ -28,7 +29,14 @@ describe('UI5 templates', () => {
                 sapux: true
             }
         });
-        fs.commit(() => 0)
         expect((fs as any).dump(projectDir)).toMatchSnapshot();
+        return new Promise((resolve) => {
+            // write out the files for debugging
+            if (debug) {
+                fs.commit(resolve);
+            } else {
+                resolve(true);
+            }
+        });
```

Comment:
This is good! Anyway we can abstract this as It seems to be required in most of the tests? Could always do it later.